perception/object_recognition/detection/tensorrt_yolo3/package.xml

```diff
@@ -1,30 +1,24 @@
 <?xml version="1.0"?>
-<package>
+<package format="3">
     <name>tensorrt_yolo3</name>
     <version>0.1.0</version>
     <description>tensorrt_yolo3</description>
 
     <maintainer email="cirpue49@yahoo.co.jp">Kosuke Murakami</maintainer>
     <license>Apache 2.0</license>
```

Comment:
Apache License 2.0

Reply:
Fixed in 419d35ab19e004b348b68bba288ad54d9905912f